.circleci/config.yml

```diff
@@ -69,6 +69,19 @@ jobs:
       - run: yarn install
       - run: ./scripts/promote
 
+  pack-and-upload-windows-installer:
+    <<: *pack-and-upload-tarballs
```

Comment:
TIL: [`<<` syntax in yaml](https://stackoverflow.com/questions/41063361/what-is-the-double-left-arrow-syntax-in-yaml-called-and-wheres-it-specced/41065222).